resolwe_bio/docker_images/legacy/packages-debian.txt

```diff
@@ -0,0 +1,56 @@
+bedtools
+samtools
+picard-tools
+bowtie
+bowtie2
+bwa
+tophat
+abyss
+# required by abyss scripts
+bsdmainutils
+p7zip-full
+python-pip
+python3-pip
+python-xlrd
+python-setuptools
+python3-setuptools
+python-requests
+python3-requests
+python-appdirs
+python3-appdirs
+python-six
+python3-six
+python-yaml
+python3-yaml
+python-wheel
+python3-wheel
+python-dev
+libpython2.7
+build-essential
+gfortran
+r-base-core
+tabix
+pypy
+# required for ea-utils
+libgsl2
+libgsl-dev
+# required for building lxml python package
+libxml2-dev
+libxslt1-dev
+# required for compiling R package arrayQualityMetrics
+libxt-dev
+libcairo2-dev
+xml2
+libcurl3
+libcurl4-openssl-dev
+trimmomatic
+texlive-latex-base
```

Comment:
Not sure, I just copied this over from the old image? We probably need it for generating some reports.